extensions/README.md

```diff
@@ -0,0 +1,391 @@
+[CoRIM
+specification](https://www.ietf.org/archive/id/draft-ietf-rats-corim-02.htm)
```

Comment:
```suggestion
The [CoRIM
data format](https://datatracker.ietf.org/doc/draft-ietf-rats-corim)
```

Reply:
I'm specificicallying linking to draft 2 here since that is what we're following; The latest version may not match.

Reply:
OK.  My comment was about using the IETF datatracker's which is supposed to have stronger URL stability.  If we need a precise version we use the versioned link https://datatracker.ietf.org/doc/draft-ietf-rats-corim/02